profiles/common/modules/features/contact_form/contact_form.install

```diff
@@ -35,4 +35,42 @@ function contact_form_disable() {
   menu_link_delete(NULL, 'contact');
   // Activation message.
   drupal_set_message(t('Contact form feature is now disabled on your site.'));
+  module_disable(array('contact'), FALSE);
```

Comment:
We should not disable modules the current module depends upon as other modules might depend from  `contact`.